tests/e2e/test_main.py

```diff
@@ -833,3 +842,59 @@ def test_data_cp_from_cloud_to_storage(
         res = cli_runner(["neuro", "rm", "-r", storage_url])
         if res.returncode != 0:
             logger.error(f"WARNING: Finalization failed! {res}")
+
+
+@pytest.fixture
+def disk(cli_runner: CLIRunner) -> Iterator[str]:
+    # Create disk
+    res = cli_runner(["neuro", "disk", "create", "1G"])
+    assert res.returncode == 0, res
+    assert res.stderr == ""
+    disk_id = None
+    for line in res.stdout.splitlines():
+        elements = line.split()
+        if len(elements) >= 3:
+            disk_id, storage, uri, *_ = elements
+            if disk_id.startswith("disk") and uri.startswith("disk:"):
+                break
+
+    if disk_id is None:
+        raise Exception("Unable to locate disk id in neuro output")
+    yield disk_id
+
+    # Delete disk
+    res = cli_runner(["neuro", "disk", "rm", disk_id])
+    assert res.returncode == 0, res
+    assert res.stderr == ""
```

Comment:
please do this in finally block , otherwise all failed tests will leave disks non-cleaned-up